core/modules/proto/worker.proto

```diff
@@ -73,6 +73,7 @@ message TaskMsg {
     required int32 jobid = 11;
     required bool scaninteractive = 12;
     required int32 attemptcount = 13;
+    optional uint32 czarid = 14;
```

Comment:
I've noticed that you have type mismatch in the `proto` definition where you use `unsigned int` versus the rest of the code where you use `int`. My understanding is that you do tis because you're making `czarid` the optional field in the protocol definition. This complicates the rest of your code. I would suggest to avoid this and make it mandatory. In the end we do have such thing as `czarId` in Qserv.
`